sentry-core/src/main/java/io/sentry/core/SentryLevel.java

```diff
@@ -2,6 +2,7 @@
 
 /** the SentryLevel */
 public enum SentryLevel {
+  LOG,
```

Comment:
@HazAT This isn't in the docs eh?
Could you please clarify what this level is?

Oh, I assume it's some JS thing leaking in? Can we leave the value be mapped as unknown and travel through the serialization/deserialization instead?

Reply:
@bruno-garcia yes, this is JS leaking in.
this is a value and not an unknown key, as we have a strong typed `enum`, it won't work, unless we add a generic value to `SentryLevel` and assume it for all the unknown values.

Reply:
`LogLevel.Log` really makes no sense imho.
But sounds like the most pragmatic way to solve this though.

Can we have a javadoc on that explain where it comes from?

Reply:
@bruno-garcia Added a javadoc